types/connect/index.d.ts

```diff
@@ -1,94 +1,163 @@
-// Type definitions for connect v3.4.0
+// Type definitions for connect 3.7
 // Project: https://github.com/senchalabs/connect
 // Definitions by: Maxime LUCE <https://github.com/SomaticIT>
 //                 Evan Hahn <https://github.com/EvanHahn>
+//                 geekley <https://github.com/geekley>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
-
 import * as http from "http";
+import connect = createServer;
 
 /**
- * Create a new connect server.
- * @public
+ * When the `connect` module is required, a function is returned that will construct a new app when called.
+ * This app will store all the middleware added and is, itself, a function.
+ *
+ * The core of Connect is "using" middleware. Middleware are added as a "stack" where incoming
```

Comment:
The first two paragraphs seem fine but why is there so much here? Is this a typical amount of documentation?